src/main/java/com/travelagency/controllers/TripController.java

```diff
@@ -60,18 +60,11 @@ public boolean deleteTrip(Long id) {
         return true;
     }
 
-    //TODO: Search trips methode ->Ismail
-    public Iterable<Trip> searchTrips(String searchInput) {
+    public Optional<List<Trip>> searchTripsName(String searchInput) {
```

Comment:
searchTripName does not cover the functionality.... it searches in a lot more places, so rename.

searchTripByKeyword... ofzoiets